pom.xml

```diff
@@ -179,24 +171,6 @@
                         <skip>true</skip>
                     </configuration>
                 </plugin>
-                <plugin>
-                    <groupId>org.codehaus.mojo</groupId>
-                    <artifactId>findbugs-maven-plugin</artifactId>
-                    <version>${version.findbugs-maven-plugin}</version>
-                    <configuration>
-                        <effort>Max</effort>
-                        <threshold>Low</threshold>
-                        <xmlOutput>true</xmlOutput>
-                        <failOnError>false</failOnError>
-                    </configuration>
-                    <executions>
-                        <execution>
-                            <goals>
-                                <goal>check</goal>
-                            </goals>
-                        </execution>
-                    </executions>
-                </plugin>
```

Comment:
Was there a particular reason to drop this (and its version above), or just a merge conflict issue? I added findbugs tag recently to test it locally same way as Jenkins did and found issues in my PRs.

Reply:
Yes, I changed to the latest Jenkins plugin pom which uses spotbugs not findbugs and already configures all of this:

https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L580